compiler/crates/react_napi/package.json

```diff
@@ -1,20 +1,23 @@
 {
-    "name": "@react/forget-napi",
```

Comment:
this seems to be the only `package.json` in the repo using "4 spaces", so it's the biggest diff. the only change should be the new repository field.

viewing the diff with "hide whitespace" makes this more clear :)